src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -99,7 +99,7 @@ public CompletionRules GetRules()
             OptionSet? options = null)
         {
             var document = text.GetOpenDocumentInCurrentContextWithChanges();
-            var languageServices = document?.Project.LanguageServices ?? _services.GetLanguageServices(Language);
+            var languageServices = document?.Project.Services ?? _services.GetLanguageServices(Language).ProjectServices;
```

Comment:
Renaming the local or no?

Reply:
generally no.  because it's just going to make these even large.  and, frankly, 'language services' isn't a name that bothers me.  it still is the thing you ask for for ILanguageService's.